src/bproxy.c

```diff
@@ -12,12 +12,12 @@ void conn_init(uv_stream_t *handle)
   conn_t *conn = malloc(sizeof(conn_t));
   conn->parser = malloc(sizeof(http_parser));
   conn->request = malloc(sizeof(http_request_t));
-  conn->proxy_handle = malloc(sizeof(conn_t));
```

Comment:
is there a reason to remove memory allocation for proxy_handle?

Reply:
In next line proxy_handle is set to NULL. That means allocation is lost and it's leaking memory.

Reply:
okay, good.